demisto_sdk/commands/pre_commit/hooks/general_hook.py

```diff
@@ -0,0 +1,6 @@
+from demisto_sdk.commands.pre_commit.hooks.hook import Hook
+
+
+class GeneralHook(Hook):
+    def prepare_hook(self, **kwargs):
```

Comment:
we can make this the default in Hook and then just use hook. we dont really need this class

Reply:
I tried using Hook, but I got an error that it is not possible to instantiate an abstract class, and I must use a class that inherits Hook.